pages/landing/hipaa/template/_product-details.html

```diff
@@ -0,0 +1,84 @@
+<h2>What exactly are we building?</h2>
```

Comment:
Given that this is the first section after the images, it probably makes more sense for this one to be in the present tense. So instead of "what are we building" it's "what you get" and instead of "our solution will support" it's "The solution includes."

Reply:
Also, the page is becoming a bit long, without clear section headings, so it's harder to following. Paging @ebeneliason for help in organizing this info!

Reply:
Switched to present tense in copy in 32bb1e8d.

Reply:
We've reviewed and iterated on the page organization together today - so I will mark this convo as resolved.